Gemfile.lock

```diff
@@ -319,4 +320,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   1.14.4
+   1.14.6
```

Comment:
As far as I know this line only matters when it comes to diagnosing problems with bundler, but I am only mildly certain of that. I am happy to revert the line, though I think that may cause even more confusion if there *were* a problem with bundler, since our lock file would be inaccurate. Thoughts?